src/@types/auth.ts

```diff
@@ -90,15 +89,20 @@ export enum SSOAction {
 }
 
 /**
- * https://spec.matrix.org/v1.7/client-server-api/#matrix-user-id
+ * A client can identify a user using their Matrix ID.
+ * This can either be the fully qualified Matrix user ID, or just the localpart of the user ID.
+ * @see https://spec.matrix.org/v1.7/client-server-api/#matrix-user-id
```

Comment:
This is certainly helpful, but for bonus points it would be nice to say where this thing is used: if I look at https://pr3537--js-sdk-docs-previews.netlify.app/stable/types/_internal_.userloginidentifier it's just somewhat lacking in context.

Anyway no need to change this now but something I think we could be better at in future.